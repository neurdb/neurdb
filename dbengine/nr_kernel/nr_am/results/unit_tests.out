SELECT run_nram_tests();
INFO:  Serialization test pass!
INFO:  Copy test pass!
 run_nram_tests 
----------------
 
(1 row)

