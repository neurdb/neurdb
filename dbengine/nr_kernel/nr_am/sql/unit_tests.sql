SELECT run_nram_tests();
