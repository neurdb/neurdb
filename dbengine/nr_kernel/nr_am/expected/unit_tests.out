SELECT run_nram_tests();
INFO:  KV serialization test pass!
INFO:  Value copy test pass!
INFO:  Rocks service basic test passed!
INFO:  Rocks service client GET PUT test passed!
INFO:  Rocks service client RANGE SCAN test passed!
INFO:  KV channel test passed!
INFO:  KV channel sequential test passed!
INFO:  KV channel multi-process test passed
INFO:  KV channel basic message test passed
 run_nram_tests 
----------------
 
(1 row)

